ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/core-2-ospf-ldp.cfg

```diff
@@ -207,13 +207,16 @@ ip route vrf MGMT 0.0.0.0/0 192.168.0.1
 mpls ip
 !
 mpls ldp
```

Comment:
Please verify that mpls ldp supports picking up the router-id from router general. Try to find a show command that shows the actual used router-id.

Reply:
```
s1-leaf1(config-mpls-ldp)#sh active 
mpls ldp
   router-id _10.255.0.1_
   transport-address interface Loopback0
   interface disabled default
   no shutdown
s1-leaf1(config-mpls-ldp)#no router-id _10.255.0.1_


s1-leaf1(config)#router general 
s1-leaf1(config-router-general)#router-id ipv4 **10.255.0.10**
s1-leaf1(config-router-general)#ex
s1-leaf1(config)#sh mpls ldp 
LDP status: enabled, operational
Local LDP Identifier: **10.255.0.10:0**
   LDP Identifier set explicitly

Reply:
yes. Local LDP id is choosing router general router id.